repositories.yaml

```diff
@@ -169,6 +169,7 @@ libraries:
       - gfx/metrics.yaml
       - toolkit/components/glean/metrics.yaml
       - toolkit/components/processtools/metrics.yaml
+      - dom/media/metrics.yaml
```

Comment:
@chutten I have added this to the Gecko library.
Other files are added to the firefox-desktop application. What's our criteria of what goes where?